technical-reports/format/design-token.md

```diff
@@ -17,6 +17,8 @@ Note: The `$type` property has been added to ensure this example is valid. Pleas
 
 </aside>
 
+The name of your design token must not include any whitespace. Otherwise your design token is considered as invalid.
```

Comment:
```suggestion
Design tokens are written in kebab-case. Otherwise your design token is considered as invalid.
```

Reply:
I think this should be left as is. Kebab-case is only one way of avoiding whitespace... `snake_case` or `CamelCase` are fine too.